src/utils/errors/ErrorCodes.ts

```diff
@@ -0,0 +1,6 @@
+export enum ErrorCodes {
+  CONFIGURER_ERROR,
```

Comment:
You could define value for each so you can get rid of that long .toString().toLowerCase() call.
```
export enum ErrorCodes {
  CONFIGURER_ERROR = 'configurer_error',
  ...
}
```